cves/CVE-2015-1872.yml

```diff
@@ -89,17 +100,28 @@ fixes_vcc_instructions: |
   The notes field is optional - place anything that clarifies things or you
   find interesting in there.
 fixes:
-- commit: fabbfaa095660982cc0bc63242c459561fa37037
-  note: ''
-- commit: ''
-  note: ''
+  - commit: fabbfaa095660982cc0bc63242c459561fa37037
+    note: 'Fix in most recent version: 2.5.4'
+  - commit: 08822122987f563b72dff77b019ae3f98b0acebb
+    note: Identified by kirti301290 version 2.4.7
+  - commit: 4c246c65bfa221c45452923cf148f7b11245b6d5
+    note: 'Backport fix version: 2.2.14'
+  - commit: fe618beccf10566e834d6a81e6e432263331553b
+    note: 'Backport fix version: 2.0.7'
+  - commit: fd6a9fcd427b86c0a249a8cdb33f746f33706725
+    note: 'Backport fix version: 1.2.12'
+  - commit: 75b0cfcf105c8720a47a2ee80a70ba16799d71b7
+    note: 'Backport fix version: 0.7.17'
 vccs:
-- commit: 254f653b24cd110fb452531bcad13083d592354e
-  note: Identified by archeogit,
-- commit: 4922a5b0ee9108e406b8b14356f19851702604d1
-  note: Identified by archeogit,Identified by SZZUnleashed
-- commit: ''
-  note: ''
+  - commit: 254f653b24cd110fb452531bcad13083d592354e
+    note: |
+      This is where support for palettized 8bpp, encoding an image by mapping
+      pixels to a pallet containing only colors that present in the image
+      stored at 8 bits per pixel. Identified by archeogit.
```

Comment:
This is much more descriptive than the original. Great improvement. Not personally familiar with what Archeogit is.